setup.py

```diff
@@ -57,7 +57,7 @@
             'qasm_file = pennylane_qiskit:load_qasm_from_file',
             ],
         },
-    'description': 'PennyLane plugin for qiskit-terra',
+    'description': 'PennyLane plugin for Qiskit',
```

Comment:
Also sneaking in an unrelated tagline update for PyPI:
![image](https://github.com/PennyLaneAI/pennylane-qiskit/assets/38584660/a2c35ca0-583e-423b-833f-fe62405d0792)